src/approaches/bilevel_planning_approach.py

```diff
@@ -89,7 +89,6 @@ def _run_sesame_plan(self, task: Task, nsrts: Set[NSRT],
             raise ApproachFailure(e.args[0], e.info)
         except PlanningTimeout as e:
             raise ApproachTimeout(e.args[0], e.info)
-
```

Comment:
remove this change (you can do `git checkout origin/master src/approaches/bilevel_planning_approach.py`)